libs/market-list/src/lib/components/landing/select-market-list.tsx

```diff
@@ -0,0 +1,113 @@
+import { Arrow, Sparkline } from '@vegaprotocol/ui-toolkit';
+import { useRouter } from 'next/router';
+import { mapDataToMarketList } from '../../utils';
+import type { MarketList } from '../__generated__/MarketList';
+
+export interface MarketSparklineProps {
+  candles?: {
+    close: string | undefined | number;
+    open: string | undefined | number;
+  }[];
+}
+
+export const MarketSparkline = ({ candles }: MarketSparklineProps) => {
+  return (
+    <Sparkline
+      width={100}
+      height={20}
+      data={
+        candles
+          ?.filter((m) => !isNaN(Number(m.close)))
+          ?.map((m) => Number(m.close)) || []
+      }
+    />
+  );
+};
+
+export interface SelectMarketListProps {
+  data: MarketList | undefined;
+}
+
+export const SelectMarketList = ({ data }: SelectMarketListProps) => {
+  const thClassNames = (direction: 'left' | 'right') =>
+    `px-8 text-${direction} font-sans font-normal text-ui-small leading-9 capitalize mb-0 text-dark/80 dark:text-white/80`;
+  const tdClassNames =
+    'px-8 font-sans leading-9 capitalize text-ui-small text-right';
+  const priceChangeClassNames = (value: number) =>
+    value === 0
+      ? 'text-black dark:text-white'
+      : value > 0
+      ? `text-green`
```

Comment:
please, gods of colors, let's change this green!  @campbellssource @PeteWilliams 

<img width="1436" alt="Screenshot 2022-05-18 at 11 49 05" src="https://user-images.githubusercontent.com/16125548/169027734-bcefc5fa-627e-411b-9c61-48bd33abdef6.png">

Reply:
Green: #008545
Red: #EB001B 
from @PeteWilliams